Preparing r151036 release notes (#2057)
omniosorg/omnios-build: doc/ReleaseNotes.md

<img src="https://omniosce.org/OmniOSce_logo.svg" height="128">
</a>

# Release Notes for OmniOS v11 r151036

Stable Release, 2nd of November 2020

`uname -a` shows `omnios-r151036-4a32ffb911`

r151036 release repository: https://pkg.omnios.org/r151036/core

## Upgrade Notes

Before upgrading, please ensure that your `package/pkg` package is up-to-date

> by running `pkg update pkg`. The `pkg` software was recently updated for all

Reviewer: ```suggestion
by running `pkg update pkg`. The `pkg` software was recently updated for all supported
```

> releases to properly handle conditional dependencies which are more widely used

Reviewer: ```suggestion
releases (r30, r32, r34) to properly handle conditional dependencies which are more widely used
```

  split out into a new libzfsutil library. One of the benefits of this is
  that the `zdb` command no longer needs root privileges for some operations.

* ZFS channel programs have gained `zfs.sync.inherit` and `zfs.check.inherit`
  to allow clearing of properties on a dataset, restoring them to the default
  of being inherited from an ancestor.

### Package Management

* `pkg rehydrate` now works properly for global zone IPS images.

* When `pkg` is unable to find a solution for an upgrade, retrying the update
  command with `-vv` will now produce additional information which may help
  with determining the underlying cause.

* The optional OmniOS Extra package repository is available and will be
  added by default for new interactive installations of OmniOS (unless
  disabled through the post-install configuration menu). It can also be added
  retrospectively to upgraded systems using:
  ```
  pkg set-publisher -O https://pkg.omnios.org/r151036/extra extra.omnios
  ```
  A [full list of packages](https://pkg.omniosce.org/r151036/extra/en/catalog.shtml) is available.

### Hardware Support

* Support for temperature, voltage and current sensors in CPUs and other

>   hardware has been added.

Reviewer: example command to query this would be nice

## Upgrade Notes

> Before upgrading, please ensure that your `package/pkg` package is up-to-date.

Reviewer: ... by runnig ...

> This was recently updated for all releases to properly handle conditional

Reviewer: ```suggestion
The `pkg` software was recently updated for all releases to properly handle conditional
```
